stack.yaml

```diff
@@ -1,9 +1,9 @@
-# This resolver file contains a ghc-8.6.5-based stackage LTS, with a
+# This resolver file contains a ghc-8.10.4-based stackage LTS, with a
 # released cardano-node version, and supporting libraries.
 #
 # NOTE: Remember to update the version matrix in README.md when
 # bumping the Cardano version.
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/1c257e85cadb503ced0fe0a9ba757ae449baaf0b/snapshots/cardano-1.25.1.yaml
+resolver: cardano-1.25.1.yaml
```

Comment:
Is it time to add the snapshot to cardano-haskell?

Reply:
Want to see if CI's green after un-pinning hspec, will move after that